spec/models/journeys/teacher_student_loan_reimbursement_spec.rb

```diff
@@ -32,4 +32,25 @@
 
     it { is_expected.to eq(Journeys::TeacherStudentLoanReimbursement::SlugSequence) }
   end
+
+  describe ".page_sequence_for_claim" do
+    let(:completed_slugs) { [:test] }
+    let(:current_slug) { [:test2] }
+    let(:claim) { double }
+
+    subject(:page_sequence) { described_class.page_sequence_for_claim(claim, completed_slugs, current_slug) }
+
+    it { is_expected.to be_a(Journeys::PageSequence) }
+
+    it "populates the page sequence attributes" do
+      expect(page_sequence.claim).to eq(claim)
+      expect(page_sequence.current_slug).to eq(current_slug)
+      expect(page_sequence.completed_slugs).to eq(completed_slugs)
+    end
+
+    it "creates a slug sequence" do
+      expect(Journeys::TeacherStudentLoanReimbursement::SlugSequence).to receive(:new).with(claim)
+      page_sequence
+    end
+  end
```

Comment:
We could probably test this method in just one place, like in a base spec, unless we think it can easily be overridden and have different behaviour for each journey. Perhaps that's what you are thinking.

Reply:
Yes I have been thinking that we can use shared examples for some of these tests, but for now I wanted to keep it simple and readable due to the heavy refactoring.